seedlet/assets/sass/blocks/cover/_editor.scss

```diff
@@ -6,6 +6,8 @@
 	margin-top: inherit;
 	margin-bottom: inherit;
 
+	@include innerblock-margin-clear(".wp-block-column");
```

Comment:
I'm not sure I'm following the use case here, but this is what it looks like when I place a paragraph inside a columns block inside a cover block: 

<img width="649" alt="Screen Shot 2021-01-13 at 1 42 02 PM" src="https://user-images.githubusercontent.com/5375500/104495097-4e816600-55a5-11eb-88f5-733d19d6e21d.png">

Should this mixin remove that top margin from the paragraph?

Reply:
I'm not sure how well I'm following all this either, but I don't think so, and it doesn't seem to. If @kjellr wouldn't mind taking a glance at this, I'd much appreciate it.

Reply:
Update: I've noticed that in viewing the post itself, there wouldn't be a top margin on those paragraphs, although there seem to be quite a few discrepancies when it comes to vertical spacing on the editor vs the post itself (both with, and without this PR), so I'm not sure how far we want to go in trying to rectify that as part of this PR.

Reply:
Gotcha. In that case, is this mixin needed at all?

Reply:
Haha. :thinking: Maybe not? I copied [this implementation from Twenty Twenty One](https://github.com/WordPress/twentytwentyone/blob/4c83b2781ae9cc9002ef8e0a5c275b2cea8a7f08/assets/sass/05-blocks/cover/_editor.scss) because I had assumed this was, for some reason, desirable.

I have to confess I couldn't actually work out exactly what this achieves on Twenty Twenty One either, but presumed I was missing something and we needed to keep it.

Reply:
If it's not needed let's leave it out. Keep it simple!

Reply:
:+1: I'll make this change.